dat/DOOM.dat

```diff
@@ -309,6 +304,14 @@ game (
 game (
 	name "Doom II - No Rest for the Living (Unity)"
 	description "Doom II - No Rest for the Living (Unity)"
+	rom ( name EXTRAS.WAD size 64779 crc 8068229f md5 0c7caf25ad1584721ff5ecc38dec97a0 sha1 55a6a95c99d7fe7b9501d34b0afecd3252a629e1 )
+	rom ( name NERVE.WAD size 3821966 crc ab68447f md5 4214c47651b63ee2257b1c2490a518c9 sha1 321e951f6cc6e9d51bc16eac53e0d001f0f3f338 )
+)
+
+game (
+	name "Doom II - No Rest for the Living (Unity) (Update - September 3)"
+	description "Doom II - No Rest for the Living (Unity) (Update - September 3)"
+	rom ( name EXTRAS.WAD size 64587 crc 82bccddc md5 38e1bb6cdd1e4227eef13d22f6e35209 sha1 0f2a7139b640e19a08f384562f907cb0734b0512 )
 	rom ( name NERVE.WAD size 3821966 crc ab68447f md5 4214c47651b63ee2257b1c2490a518c9 sha1 321e951f6cc6e9d51bc16eac53e0d001f0f3f338 )
```

Comment:
Note that RetroArch's RDBs take One-Rom-Per-Game. We'll need to target just one unique entry per game.

Reply:
In DOOM.DAT alone, Final Doom-Ejection and Strife-Veteran Edition include multiple WADs.
In the first place, EXTRAS.WAD is a PWAD that does not work by itself like VOICE.WAD, so it should be set with IWAD such as DOOM.DAT.
However, if IWAD is missing, Prboom adds it.
For example, Doom II - No Rest for the Living is PWAD, but at runtime it also loads DOOM2.WAD that is not specified in the RDB.
All you have to do is to make the appearance of RDB. Would you like to do the other things like CloneROM?

Reply:
I'll merge and re-build the RDBs. Mind testing once they're in?